packages/kit-headless/src/components/qwik-ui-provider/qwik-ui-provider.tsx

```diff
@@ -12,7 +12,7 @@ import {
   type Signal,
   Fragment,
 } from '@builder.io/qwik';
-import { type JSXNode } from '@builder.io/qwik/jsx-runtime';
+import { type JSXNode } from '@builder.io/qwik';
```

Comment:
Let's just remove this file, and anywhere it's being used. It's an old impl

Reply:
Alright